pkg/resource/cluster.go

```diff
@@ -38,6 +38,7 @@ const (
 	CrdbContainerImageAnnotation = "crdb.io/containerimage"
 	CrdbVersionAnnotation        = "crdb.io/version"
 	CrdbHistoryAnnotation        = "crdb.io/history"
+	CrdbCAExpirationAnnotation   = "crdb.io/certexpiration"
```

Comment:
One thing I noticed is that crdb.io does not exist. @keith-mcclellan, which domain do we want on these?

Reply:
why do we have a url here at all @chrislovecnm ?

Reply:
@keith-mcclellan if the prefix is omitted, the annotation Key is presumed to be private to the user. Automated system components (e.g. kube-scheduler, kube-controller-manager, kube-apiserver, kubectl, or other third-party automation) which add annotations to end-user objects must specify a prefix. Please see k8s documentation for annotations: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
got it thanks